mx.py

```diff
@@ -10694,6 +10694,7 @@ def __call__(self, msg):
              '-doclet', 'org.apidesign.javadoc.codesnippet.Doclet',
              '-docletpath', snippetslib,
              '-snippetpath', snippets,
+             '-verifysincepresent',
```

Comment:
The `mx javadoc --unified` is used to produce documentation for API - probably only by Truffle. As far as I understand `mx javadoc` is used for documentation for the code - to help people orient around the project source. Probably the `-verifysincepresent` check is only useful for the documentation of an API and would be annoying otherwise.